backend/song/tests/test_views.py

```diff
@@ -0,0 +1,272 @@
+import pytest
+
+from django.urls import reverse
+
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_201_CREATED,
+    HTTP_204_NO_CONTENT,
+    HTTP_403_FORBIDDEN,
+)
+
+from backend.song.models import Artist
+
+
+pytestmark = pytest.mark.django_db
+
+
+class TestCategoryView:
+
+    def test_create_category(self, auth, create_category):
+
+        assert create_category.status_code == HTTP_201_CREATED
+        assert create_category
+
+    def load_category_list(self, create_category, get):
+        response = get(reverse('song:categories-list'))
+
+        assert response.status_code == HTTP_200_OK
+        assert response.data
+        assert response.data['results']
+
+    def test_load_category_list_filtered(self, create_category, get):
+        response = get(f"{reverse('song:categories-list')}?category_name=rumba")
+
+        assert response.status_code == HTTP_200_OK
+        assert response.data['results'][0]['category_name'] == 'rumba'
+        assert response.data['results']
+
+    def test_load_category_detail(self, create_category, get):
+        response = get(reverse('song:categories-detail', args=[1]))
+
+        assert response.status_code == HTTP_200_OK
+        assert response.data['category_name'] == 'rumba'
+
```

Comment:
the same comment apply here we need to check if this have all the field we want in the response...